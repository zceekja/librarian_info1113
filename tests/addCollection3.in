add book book1.csv 111111
add collection book1.csv