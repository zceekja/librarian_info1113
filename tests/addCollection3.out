user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: 16 books successfully added.

user: 